it/src/test/java/com/linecorp/centraldogma/it/mirror/git/GitMirrorTest.java

```diff
@@ -419,6 +419,14 @@ void cannotMirrorToInternalRepositories(String localRepo) {
                 .hasMessageContaining("invalid localRepo:");
     }
 
+    @Test
+    void remoteToLocalEmptyMessage() throws Exception {
+        pushMirrorSettings("/local", "/remote", null);
+        final RevCommit call = git.commit().setMessage("").call();
+        mirroringService.mirror().join();
+        assertThat(call.getFullMessage()).isEmpty();
+    }
```

Comment:
This test you wrote basically does the following:

1. Add a commit to the remote Git repository.
2. Run mirroring.
3. You check the commit message of the commit you pushed in (1).

As you might have noticed, the step (3) doesn't make sense. You have to check the local Central Dogma repository has the commit that contains the message copied from the commit you added in the step (1), by using `client`.

Try to understand the test `remoteToLocal()` first.

Reply:
I updated the test